mmv1/templates/terraform/examples/spanner_instance_partition_basic.tf.tmpl

```diff
@@ -0,0 +1,14 @@
+resource "google_spanner_instance" "main" {
```

Comment:
Can you give the instance a name to avoid the randomness of `instanceId`? If name is provided, it will be used for `instanceId`.
```
name         = "{{index $.Vars "partition_name"}}"
```